value-fixture/test/org/immutables/fixture/jdkonly/JdkOnlyTest.java

```diff
@@ -48,10 +48,10 @@ public void collections() {
         .addPols(RetentionPolicy.RUNTIME, RetentionPolicy.RUNTIME)
         .build();
 
-    check(coll.ints()).isOf(1, 2, 3, 4, 5, 6);
-    check(coll.navs()).isOf(3, 2, 1);
-    check(coll.ords()).isOf(4, 5, 6, 7, 8, 9);
-    check(coll.pols()).isOf(RetentionPolicy.RUNTIME);
+    check(coll.ints()).hasContentInAnyOrder(1, 2, 3, 4, 5, 6);
+    check(coll.navs()).hasContentInAnyOrder(3, 2, 1);
+    check(coll.ords()).hasContentInAnyOrder(4, 5, 6, 7, 8, 9);
+    check(coll.pols()).hasContentInAnyOrder(RetentionPolicy.RUNTIME);
```

Comment:
To make comparisons stable between JDK8 and JDK9+.
`isOf` was taking verifying elements is exact order